src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -291,6 +291,11 @@ public enum NuGetLogCode
         /// </summary>
         NU1801 = 1801,
 
+        /// <summary>
+        /// MacCatalyst platform fell back to xamarin.ios
+        /// </summary>
+        NU1901 = 1901,
```

Comment:
This should be NU1703, given that compat warnings are NU17XX 
Log codes have a pattern.